src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinue/EditAndContinueTestBase.vb

```diff
@@ -181,77 +181,117 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
             Return MetadataTokens.Handle(table, rowNumber)
         End Function
 
-        Friend Shared Sub CheckEncLog(reader As MetadataReader, ParamArray rows As EditAndContinueLogEntry())
-            AssertEx.Equal(rows, reader.GetEditAndContinueLogEntries(), itemInspector:=AddressOf EncLogRowToString)
-        End Sub
-
-        Friend Shared Sub CheckEncLogDefinitions(reader As MetadataReader, ParamArray rows As EditAndContinueLogEntry())
-            AssertEx.Equal(rows, reader.GetEditAndContinueLogEntries().Where(Function(entry) IsDefinition(entry.Handle.Kind)), itemInspector:=AddressOf EncLogRowToString)
-        End Sub
-
         Friend Shared Function IsDefinition(kind As HandleKind) As Boolean
```

Comment:
>Friend [](http://example.com/codeflow?start=8&length=6)

Helpers ported from C#. We should try move them to common code to avoid duplication in future.